lib/modules/platform/pr-body.ts

```diff
@@ -11,6 +16,7 @@ const reviewableRegex = regEx(/\s*<!-- Reviewable:start -->/);
 
 export function hashBody(body: string | undefined): string {
   let result = body?.trim() ?? '';
+  result = result.replace(prDebugDataRe, '');
```

Comment:
so we'll never update debug data until any dependency or other config change. 🤔

Reply:
The alternative is updating each PR on almost every run?

Reply:
We will update debug data  on every time we call `getPrBody` regardless 

do you want to have another trigger for it? i dont get it

and im not sure how we hash the body, and why we should hash it without the `prDebugData`, that was suggested by @zharinov 

im not sure what to do here